pkgs/applications/audio/cozy-audiobooks/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv, fetchFromGitHub
+, ninja
+, boost
+, meson
+, pkgconfig
+, wrapGAppsHook
+, appstream-glib
+, desktop-file-utils
+, gtk3
+, glib
+, gst_all_1
+, gobjectIntrospection
+, python3Packages
+, file
+, cairo
+, sqlite
+, gettext
+, gnome3
+}:
+
+stdenv.mkDerivation rec {
+  name = "cozy-${version}";
+  version = "0.6.3";
+
+  src = fetchFromGitHub {
+    owner = "geigi";
+    repo = "cozy";
+    rev = version;
+    sha256 = "0xs6vzvmx0nvybpjqlrngggv2x8b2ky073slh760iirs1p0dclbc";
+  };
+
+  postPatch = ''
+    chmod +x data/meson_post_install.py
+    patchShebangs data/meson_post_install.py
+    substituteInPlace cozy/magic/magic.py --replace "ctypes.util.find_library('magic')" "'${file}/lib/libmagic${stdenv.hostPlatform.extensions.sharedLibrary}'"
+  '';
+  postInstall = ''
+      wrapProgram $out/bin/com.github.geigi.cozy \
+      --prefix PYTHONPATH : "$PYTHONPATH:$(toPythonPath $out)"
+      ln -s $out/bin/com.github.geigi.cozy $out/bin/cozy-audiobooks
+  '';
+  wrapPrefixVariables = [ "PYTHONPATH" ];
+
+
+  nativeBuildInputs = [
+    meson ninja pkgconfig
+    wrapGAppsHook
+    appstream-glib
+    desktop-file-utils
+    gobjectIntrospection
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    cairo
+    gettext
+    gnome3.defaultIconTheme
+    gnome3.gsettings-desktop-schemas
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-plugins-good
+    gst-plugins-ugly
+    gst-plugins-base
+  ]) ++ (with python3Packages; [
+    python
+    gst-python
+    pygobject3
+    dbus-python
+    mutagen
+    peewee
+    magic
+  ]);
+
+  checkPhase = ''
+    ninja test
+  '';
+
+  meta = with stdenv.lib; {
+    description = ''
+       A modern audio book player for Linux using GTK+ 3
+    '';
+    maintainers = [ maintainers.makefu ];
+    license = licenses.mit;
```

Comment:
I think this should be `mit` and `gpl3` because of [this](https://github.com/geigi/cozy/blob/master/LICENSE) and [this](https://github.com/geigi/cozy/blob/master/cozy/magic/LICENSE)